From 185897a96c4ad077965d3b35e7ef0bf06ec6ff51 Mon Sep 17 00:00:00 2001
From: John Shao <johnshao@google.com>
Date: Wed, 28 Apr 2021 19:48:40 +0000
Subject: [PATCH] Hide non system overlays in ContactSelectionActivity

Bug: 182809425
Test: build and install
Change-Id: I84c187eabe14cc8694110b2c6cb7502586511b22
(cherry picked from commit ea3e6cdd4152b5878121698f3dde6542b897c191)
---
 .../android/contacts/activities/ContactSelectionActivity.java  | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/com/android/contacts/activities/ContactSelectionActivity.java b/src/com/android/contacts/activities/ContactSelectionActivity.java
index 57669faaa..2057c9d69 100644
--- a/src/com/android/contacts/activities/ContactSelectionActivity.java
+++ b/src/com/android/contacts/activities/ContactSelectionActivity.java
@@ -107,6 +107,9 @@ public class ContactSelectionActivity extends AppCompatContactsActivity implemen
     protected void onCreate(Bundle savedState) {
         super.onCreate(savedState);
 
+        getWindow().addSystemFlags(android.view.WindowManager.LayoutParams
+            .SYSTEM_FLAG_HIDE_NON_SYSTEM_OVERLAY_WINDOWS);
+
         RequestPermissionsActivity.startPermissionActivityIfNeeded(this);
 
         if (savedState != null) {
-- 
2.17.1

